src/frontend/package.json

```diff
@@ -110,6 +110,7 @@
     "redux": "4.1.2",
     "regenerator-runtime": "0.13.9",
     "reselect": "4.1.4",
+    "resize-observer-polyfill": "1.5.1",
```

Comment:
After some reflexion we (@jbpenrath and I) think that using this polyfill is not necessary. It will obly cover `opera mini` usage. We think using the native `resize observer` is better.

Reply:
To test it without the polyfill, you can use an indirection by extracting `ResizeObserver` from window in `utils/winsow.ts` like this: 

```
export const { converse, ResizeObserver } = window;
```

And then import it in you component `import { ResizeObserver } from 'utils/window`. Doing this you can easily mock it in your tests.